cmd/limactl/shell.go

```diff
@@ -41,6 +41,10 @@ func newShellCommand() *cobra.Command {
 	return shellCmd
 }
 
+// Environment variable that allows configuring the command (alias) to execute
+// in place of the 'ssh' executable.
+const envShellSSH = "LIMASHELL_SSH"
```

Comment:
```suggestion
const envShellSSH = "SSH"
```

I think this is safe and probably compatible with other applications that support custom ssh command